static/css/style.css

```diff
@@ -186,3 +186,8 @@ main {
   -moz-column-gap: 40px;
   column-gap: 40px;
 }
+
+.modal-body {
```

Comment:
Maybe don't add this CSS, but rather used built in class in bootstrap. You can find it in docs: https://getbootstrap.com/docs/4.3/components/modal/

Reply:
Thanks! I just changed it.